УП Практикум/Семинар 2/Задача 1/Задача 1.cpp

```diff
@@ -0,0 +1,16 @@
+﻿#include <iostream>
+
+int main()
+{
+    int year;
+    std::cin >> year;
+
+    if (year % 4 == 0)
```

Comment:
What is a leap year? To be a leap year, the year number must be divisible by four – except for end-of-century years, which must be divisible by 400. This means that the year 2000 was a leap year, although 1900 was not. 2024, 2028, 2032 and 2036 are all leap years.